@popperjs/core: Command failed

yarn 1.22.5

---

I update packages every day with this command:

```
yarn upgrade
```

But today I got a problem.

As before, I tried to update packages. I ran the command I showed above. As a result, I got this:

```
warning "webpack-dev-server > webpack-dev-middleware@3.7.2" has unmet peer dependency "webpack@^4.0.0".
[4/4] 🔨  Building fresh packages...
[1/5] ⠂ @popperjs/core
[2/5] ⠂ fsevents
[3/5] ⠂ core-js
[4/5] ⠂ node-sass
error /Users/afuno/app1/node_modules/@popperjs/core: Command failed.
Exit code: 127
Command: patch-package
Arguments:
Directory: /Users/afuno/app1/node_modules/@popperjs/core
```

I deleted the yarn.lock file and node_modules directory. And then I ran this command:

```
yarn install
```

And it didn't help.

Same here: v2.5.0 is breaking the install/upga:

```
❯ npm install @popperjs/core

> @popperjs/core@2.5.0 postinstall ~/Downloads/poppertest/node_modules/@popperjs/core
> patch-package

sh: patch-package: command not found
npm WARN poppertest@1.0.0 No description
npm WARN poppertest@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! @popperjs/core@2.5.0 postinstall: `patch-package`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the @popperjs/core@2.5.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/package.json
+++ b/package.json
@@ -44,7 +44,7 @@
     "build:dev": "NODE_ENV=dev babel src -d tests/visual/dist",
     "build:flow": "flow-copy-source --ignore \"**/*.test.js\" src lib && replace-in-files --string=__DEV__ --replacement=false 'lib/**/*.flow'",
     "build:typescript": "rimraf dist/typescript; flow-to-ts \"src/**/*.js\" --write --inline-utility-types; tsc-silent --project .config/tsconfig.json --createSourceFile .config/createSourceFile.js --suppress @; rimraf \"src/**/*.ts\"",
-    "postinstall": "patch-package",
+    "prepare": "patch-package",
     "prepublishOnly": "yarn build"
   },
   "prettier": {

